### PR TITLE
React docs v1

### PR DESCRIPTION
**Pull request by daverau-optimizely**
Just added the table of contents and quickie Nav component.

This branch addresses nearly all our v1 issues listed in #431

### CODE REVIEW

File: docs/index-old.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<html>
```
Comment (danoc) [inline docs/index-old.html]: Let's remove this file.

File: docs/index.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+          href: '#' + component.index.name,
+        }
+      );
+    });
+
+    let componentNodes = componentsArray.map(function(component) {
+      let json = componentsJSON['src/components/' + component.index.name + '/index.js'];
+      let desc = doctrine.parse(json.description);
+
+      // desc.tags.map(function(i){
```
Comment (danoc) [inline docs/index.jsx]: Let's remove this if not needed.

File: src/components/ArrowsInline/index.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import React from 'react';
  * Small arrows as needed next to navigational elements. The arrows will
  * inherit the color of the parent's color.
  * @param {Object} props - Properties passed to component
+ * @example <ArrowsInline>
```
Comment (danoc) [inline src/components/ArrowsInline/index.js]: Let's undo this since it's not being used.

**Comment by danoc** [comment]
It's looking good! We just have some code to remove that isn't needed anymore.

**Comment by danoc** [comment]
LGTM once build passes.

**Comment by daverau-optimizely** [comment]
Meeeeeeerge!

**Pull request merged**